Formula/boringtun.rb

```diff
@@ -1,11 +1,16 @@
 class Boringtun < Formula
   desc "Userspace WireGuard implementation in Rust"
   homepage "https://github.com/cloudflare/boringtun"
-  url "https://github.com/cloudflare/boringtun/archive/v0.3.0.tar.gz"
-  sha256 "1107b0170a33769db36876334261924edc71dfc1eb00f9b464c7d2ad6d5743d3"
+  url "https://static.crates.io/crates/boringtun/boringtun-0.4.0.crate"
```

Comment:
Can we link to the upstream issue here?

Reply:
You mean add as a code comment?

Reply:
Yes.

Reply:
actually they just tagged the release, gonna switch back to the source tarball.

Reply:
Good timing. :)